run.py

```diff
@@ -35,6 +35,6 @@
 # Commands, etc
 bot.load_extension("bot.cogs.snakes")
 
-bot.run(os.environ.get("BOT_TOKEN"))
+bot.run(os.environ.get('BOT_TOKEN'))
```

Comment:
Haha, come on, no need to change the quote marks.

Reply:
i use single quotes, just that i accidentally did the config.json stuff before so when i changed it back i naturally used single quotes